model/src/test/java/com/basistech/rosette/dm/AnnotatedTextTest.java

```diff
@@ -71,11 +71,12 @@ public void testTranslatedData() {
         ListAttribute.Builder<TranslatedData> translatedDataBuilder =
             new ListAttribute.Builder<>(TranslatedData.class);
 
+        TextDomain sourceDomain = new TextDomain(ISO15924.Latn, LanguageCode.ENGLISH, TransliterationScheme.NATIVE);
```

Comment:
Couple small things about this test:
- it might be nice to use both `getDomain()` and `getTargetDomain()` somewhere in the tests, since for now we expect both to work
- similarly, we can test both a `TranslatedData` with a source domain and one without one
- In addition to testing the contents of the target domain, we should be testing the contents of the source domain (if it wasn't specified, it should just be null)

Reply:
Good ideas. For the constructors I already set some up in `JsonTest` but that was not enough. I went ahead and added your suggested changes.